openreview/venue_request/process/error_status_process.py

```diff
@@ -0,0 +1,13 @@
+def process(client, note, invitation):
+    GROUP_PREFIX = ''
+    SUPPORT_GROUP = GROUP_PREFIX + '/Support'
+
+    forum_note = client.get_note(note.forum)
+    subject = f'''{note.content['title']} for service: {forum_note.content['title']}'''
+    message = f'''A comment was posted to your service request. 
```

Comment:
Given you are going to have a process function per invitation you can customize the email text like: 'An error occurred while running the stage... for your service request.'